datawave-monolith-umbrella/Chart.lock

```diff
@@ -1,24 +1,24 @@
 dependencies:
 - name: dwv-audit
   repository: file://../audit
-  version: 1.0.0
+  version: 1.0.0-SNAPSHOT
```

Comment:
Temporary. Once this PR is merged, each sub chart will be released, and then this will be replaced with the oci repo and actually released version